tests/infra/remote.py

```diff
@@ -614,7 +627,31 @@ def __init__(
                 ]
 
             if self.quote:
-                cmd.append(f"--quote-file={self.quote}")
+                cmd += [f"--quote-file={self.quote}"]
+
+            if start_type == StartType.start:
+                cmd += [
+                    "start",
+                    f"--member-certs={members_certs}",
+                    f"--user-certs={users_certs}",
+                    f"--gov-script={os.path.basename(gov_script)}",
+                ]
+                data_files += [members_certs, users_certs, os.path.basename(gov_script)]
+                if app_script:
+                    cmd += [f"--app-script={os.path.basename(app_script)}"]
+                    data_files += [os.path.basename(app_script)]
+            elif start_type == StartType.join:
+                cmd += [
+                    "join",
+                    "--network-cert-file=networkcert.pem",
+                    f"--target-rpc-address={target_rpc_address}",
+                ]
+                data_files += ["networkcert.pem"]
+            elif start_type == StartType.recover:
+                cmd += ["recover"]
+                # Starting a CCF node in recover does not require any additional arguments
+            else:
+                raise ValueError("CCFRemote start type should be start or join")
```

Comment:
Or recover.

Reply:
Minor: This should print the `start_type` that was passed. Could also print all the enum values automatically (`', '.join(e.name for e in StartType)`) but then if we add a new value and forget to support it here we get a very confusing error message.